pages/common/sha256sum.md

```diff
@@ -13,3 +13,7 @@
 - Read a file of SHA256 sums and verify all files have matching checksums:
 
 `sha256sum -c {{filename.sha256}}`
+
+- Verify all files have matching chesksums but don't print OK for succefully verified file:
```

Comment:
Maybe this is better:
```suggestion
- Verify all files have matching chesksums but don't show informative messages:
```

Reply:
Do you think it's clear that `informative messages` means `OK` on successful file verification

Reply:
@sbrl ?

Reply:
Hrm. What about this?
```suggestion
- Only show a message for files for which verification fails:
```